### PR TITLE
Increase Capacity How To

### PR DESCRIPTION
**Pull request by johnvincentcorpuz**

### CODE REVIEW

File: docs/group_howto/ght_capacity_increase.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+How to Increase Your Capacity or Number of Services
+====================================================
+
+The number of plans you can create is limited by the service limits set in your organization. For example, free users are only able to create free service plans. For enterprise users, your organization may also be limited in the number of services that can be created.
+
+
+Requesting a Service Limit Increase from PubSub+ Cloud Console
+---------------------------------------------------------------
+
+1. Login into Solace Cloud
```
Comment (tzoght) [inline docs/group_howto/ght_capacity_increase.rst]: Add a link  to Solace Cloud

File: docs/group_howto/ght_capacity_increase.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+2. Click Account on the top menu, and the Service Limits tab under Account Details
+
+.. image:: ../img/ght_nav_manage_service_limits.jpg
```
Comment (tzoght) [inline docs/group_howto/ght_capacity_increase.rst]: Fix the issue with image highlight

File: docs/group_howto/ght_capacity_increase.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+   :target: https://console.solace.cloud/account-details/service-limits
+
+3. On the service limit table, you can click on "Request Service Increase" for each plan of interest.
+
+.. image:: ../img/ght_request_service_increase.jpg
+
+4. A dialog box will then show up, you may then specify the number of services you require, and any accompanying details.
```
Comment (tzoght) [inline docs/group_howto/ght_capacity_increase.rst]: and add comments in "Details For Requests"

File: docs/group_howto/ght_capacity_increase.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Click on "Send Request".
+
+.. image:: ../img/ght_service_limit_increase_dialog.jpg
+
+Our product support team should receive your request, and will follow up with you the soonest.
```
Comment (tzoght) [inline docs/group_howto/ght_capacity_increase.rst]: reword the message

File: docs/group_howto/ght_capacity_increase.rst (hunk 2 above)
Comment (johnvincentcorpuz) [reply docs/group_howto/ght_capacity_increase.rst]: changed

**Pull request merged**